years.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
```

Comment:
From issue [teamed/qulice#583](https://github.com/teamed/qulice/issues/583) 
`There should be test added to check whether copyright is 2011-current year. It should check at least .java, .xml, .vm, .groovy, .txt, .fml, .properties extensions.`
No .sh file in the list.